include/votca/tools/cubicspline.h

```diff
@@ -99,6 +99,20 @@ class CubicSpline : public Spline
     void AddToFitMatrix(matrix_type &A, double x,
             int offset1, int offset2=0, double scale=1);
 
+    /**
+     * \brief Add a point (one entry) to fitting matrix
+     * \param pointer to matrix [in] [out]
```

Comment:
reference to matrix not pointer, but I am not sure how consistent the rest is.

Reply:
The rest is also not more consistent

Reply:
Yeah I know, but it is still wrong @junghans

Reply:
?

Reply:
Well, make at least the next comments correct!